<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1615</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIFT Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Browse template ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle_cam_button">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>620</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Enable camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="template_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>660</width>
      <height>495</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="live_image_label">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>50</y>
      <width>660</width>
      <height>495</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1615</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
